internal/pkg/template/templates/addons/s3/env/cf.yml

```diff
@@ -0,0 +1,54 @@
+Parameters:
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The name of the environment being deployed.
+
+Resources:
+  {{logicalIDSafe .Name}}Bucket:
+    Metadata:
+      'aws:copilot:description': 'An Amazon S3 bucket to store and retrieve objects for {{.Name}}'
```

Comment:
```suggestion
      'aws:copilot:description': 'An Amazon S3 bucket to store and retrieve objects'
```
It doesn't feel `for {{.Name}}` needs to be there.

Reply:
I wanted to keep `{{.Name}}` in it so that when people look at the progress tracker they see the name that they put in for the storage! Also, it'd be helpful for people to differentiate between the buckets if they have several s3 buckets as addons. I rephrase the description a little bit with `{{.Name}}` kept!

Reply:
Let's say the bucket name is "myBucket". `'An Amazon S3 bucket to store and retrieve objects for myBucket'` doesn't make sense to me since "myBucket" is not technically a pattern name but a bucket name. When doing `storage init` we prompt them to input the name we ask for the bucket name as well.

I feel like this sounds more like a general question about showing the logical resource name instead of putting the name into the description. Or maybe rephrase it to `"An Amazon S3 bucket named {{.Name}} to store and retrieve objects"`? TBH "to store and retrieve objects" sounds redundant too...

Reply:
nvm let's keep it as is for now. I don't think it is a big one.

Reply:
I will keep it as is then